docs/concepts.rst

```diff
@@ -271,3 +271,91 @@ Modules not yet Supported for Criteria
 :mod:`RunHistory <cbc_sdk.audit_remediation.base.RunHistory>`
 :mod:`Event <cbc_sdk.enterprise_edr.base.Event>`
 :mod:`Process <cbc_sdk.enterprise_edr.base.Process>`
```

Comment:
With our most recent updates, the Event and Process modules now support criteria

Reply:
Added them.